### PR TITLE
Performance comparison of some frameworks.

### PR DESCRIPTION
**Pull request by gongweibao**

### CODE REVIEW

File: benchmark/cluster/network/README.md [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+<td>  </td>
+<td> </td>
+<td> </td>
+<td> </td>
+<td> </td>
+</tr></tbody>
+</table>
+
+
+## Conclution
```
Comment (typhoonzero) [inline benchmark/cluster/network/README.md]: Need to add the unit of this figure

**Comment by panyx0718** [comment]
I notice the code used for testing grpc is using Paddle's threadpool. Can you change to use pthread similar to the brpc experiment. Also, it would be great to simplify your grpc experiment to be as similar to brpc experiment as possible

**Comment by panyx0718** [comment]
Could there be extra over head to create threads in threads?
https://github.com/gongweibao/tests/blob/develop/grpc_test/client.cc#L172

**Comment by panyx0718** [comment]
https://github.com/gongweibao/tests/blob/develop/grpc_test/client.cc#L165
compared with brpc experiment, which creates channel outside timing zone, grpc experiment is creating channel within timing zone

**Comment by panyx0718** [comment]
https://github.com/Tuvie/brpc/blob/master/benchmark/client.cpp#L49
brpc seems to be using sync call.

https://github.com/gongweibao/tests/blob/develop/grpc_test/client.cc#L71
while grpc is using async with maximum of 3 nested levels of threads.

**Comment by gongweibao** [comment]
It seems that they are not the key problem of performance.
It open an ISSUE in GRPC some day ago:https://github.com/grpc/grpc/issues/15348

**Pull request closed without merging**